src/locales.json

```diff
@@ -242,5 +239,36 @@
     "fr": {
       "pageTitle": "Mentions légales et contact"
     }
+  },
+  "wohnen": {
+    "de": {
+      "values": {
+        "runningServices": {
+          "heating": "Heizung",
+          "water": "Wasser/Abwasser",
+          "garbage": "Müllabfuhr",
+          "chimney": "Kaminkehrer",
+          "other": "Sonstiges"
+        },
+        "additionalServices": {
+          "garage": "Garage/Stellplatz",
```

Comment:
Is this from Neuburg? Otherwise we could rename this to Parkplatz.

Reply:
That's from neuburg.